docs/pages/admin-guides/access-controls/access-request-plugins/datadog-hosted.mdx

```diff
@@ -0,0 +1,204 @@
+and add the `editor-reviewer` role.
+
+Next, create a user called `myuser` who has the `editor-requester` role. Later in
+this guide, you will create an Access Request as this user to test the Datadog
+plugin. To this user, navigate to **Management -> Access -> Users**. Then select
+**Enroll Users** and create a user with the `editor-requester` role.
+
+You should end up with two users that look like this:
+
+![Teleport Users](../../../../img/enterprise/plugins/datadog/teleport-users.png)
+
+## Step 3/5. Generate Datadog credentials
+
+The Access Request plugin for Datadog receives Access Request events from the Teleport
+Auth Service, formats them into Datadog incidents, and sends them to the Datadog
+API. For this to work, you must provide a valid Datadog API and Application key
+to the Teleport plugin.
+
+### Generate a Datadog API key
+
+First, in the Datadog web UI navigate to **Organization Settings -> Access -> API Keys**
+to view your organization's API keys. To generate a new API key, click on **New Key**
+and name it "Teleport Access". Copy the API key to paste in a later step.
+
+![Datadog dashboard for API Keys](../../../../img/enterprise/plugins/datadog/dashboard-api-keys.png)
+
+### Create a Datadog Service Account
+
+Next, navigate to **Organization Settings -> Identity & Accounts -> Service Accounts**
+to view your organization's service accounts. To create a new service account, click
+on **New Service Account** and name it "Teleport Access". You can provide whatever
+email you'd like to associate with the service account, then assign the role
+"Datadog Standard Role". This role provides the required permissions necessary to
+manage the Teleport Access Request incidents.
+
+![Datadog dashboard for Service Accounts](../../../../img/enterprise/plugins/datadog/dashboard-service-accounts.png)
+
+Now click on the "Teleport Access" service account that was just created and click
+on the **Application Keys** tab and click on **New Key** and name it "Teleport Access"
+to create a new Application key. Copy the Application key to paste in a later step.
+
+![Datadog dashboard for Application Keys](../../../../img/enterprise/plugins/datadog/dashboard-application-keys.png)
+
+## Step 4/5. Enroll the Datadog Incident Management plugin
+
+At this point, you're now ready to enroll the Datadog Incident Management plugin.
+Navigate to **Access Management -> Enroll New Integration -> Datadog**.
+
+![Select enrollment](../../../../img/enterprise/plugins/datadog/select-enrollment.png)
+
+Provide the API and Application keys generated above. Select the desired API endpoint.
+Then provide the Datadog team handle, that you created earlier, as the fallback recipient.
+This should be "teleport-access".
+
+The fallback recipient will be the default recipient for notifications. The recipient
+can be a Datadog user email, or a Datadog team handle. You can configure more custom
+notification routing rules afterwards using Access Monitoring Rules.
```

Comment:
A user not familiar with Teleport will ask themselves "what is an AMR?", so this would benefit from a link. We don't have a nice "What is an AMR" page yet, so the most apropriate would be the AMR guide you already linked in the "Next steps".